app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_statistics.html

```diff
@@ -24,7 +25,9 @@ <h2 class="h3">Statistics per case</h2>
                     <th>Image name</th>
                     <th>Total score / max score</th>
                     <th>Average score</th>
-                    <th>Ground truth</th>
+                    {% for question in object.statistics.questions %}
```

Comment:
In `results_list.html` there is some js for hiding multiple columns at once, we will probably need that here in the future.

Reply:
Added an issue for this as well: https://github.com/comic/grand-challenge.org/issues/1155